IsUserInActiveSubsetActivity/index.ts

```diff
@@ -0,0 +1,44 @@
+﻿import { createTableService } from "azure-storage";
+
+import { getIsUserATestUser } from "../utils/checkTestUsers";
+import { getConfigOrThrow } from "../utils/config";
+import { createActivity } from "../utils/durable/activities";
+import { getIsInActiveSubset } from "../utils/featureFlags";
+
+import {
+  ActivityBodyImpl,
+  ActivityInput,
+  activityResultSuccessWithValue,
+  ActivityResultSuccessWithValue,
+  getActivityBody
+} from "./handler";
+
+const config = getConfigOrThrow();
+
+const tableService = createTableService(
+  config.BETA_USERS_STORAGE_CONNECTION_STRING
+);
+
+export {
+  ActivityBodyImpl,
+  ActivityInput,
+  ActivityResultSuccessWithValue,
+  activityResultSuccessWithValue
+};
+
+export const activityName = "IsUserInActiveSubsetActivity";
+
+const activityFunction = getActivityBody(
+  getIsInActiveSubset(
+    getIsUserATestUser(tableService, config.BETA_USERS_TABLE_NAME)
+  )
+);

```

Comment:
Alternative:
```ts
const activityFunction = pipe(
  ([t, n]) => getIsUserATestUser(t, n),
  getIsInActiveSubset,
  getActivityBody
)([tableService, config.BETA_USERS_TABLE_NAME]);
```
(not sure if it's more readable, though)

Reply:
Not sure, but please consider that in next iteration we'll pass `getIsInActiveSubset` a second function parameter, the `getIsUserACanaryUser` one, so maybe in this case using `pipe` will become not so readable. Don't know, I'll try in IP-94